src/index.html

```diff
@@ -7,12 +7,74 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute in the link tag should point to a CSS file, not an SCSS file. Make sure to compile your SCSS into CSS and link to the resulting CSS file.